client/gutenberg/extensions/publicize/gutenberg-store.jsx

```diff
@@ -75,21 +77,14 @@ const publicizeStore = {
 		 *
 		 * Updates component state in response to request finishing.
 		 *
-		 * @param {string} resultString JSON encoded result of connection request
+		 * @param {string} result Result of connection request
 		 * @return {Object} action type and (maybe) decoded JSON connection list
```

Comment:
Perhaps we could remove `(maybe) decoded JSON` now.

Reply:
Aaah true